src/cargo/core/compiler/custom_build.rs

```diff
@@ -410,6 +410,7 @@ fn build_work(cx: &mut Context<'_, '_>, unit: &Unit) -> CargoResult<Job> {
                 // If we're opting into backtraces, mention that build dependencies' backtraces can
                 // be improved by requesting debuginfo to be built, if we're not building with
                 // debuginfo already.
+                #[allow(clippy::disallowed_methods)]
```

Comment:
This should be read from config. I'll change this if people agree on.

Reply:
Don't other tools respect `RUST_BACKTRACE`, like `rustc`?  If so, wouldn't that cause inconsistent behavior to read from config?